app/AndroidManifest.xml

```diff
@@ -284,8 +284,7 @@
             android:windowSoftInputMode="adjustResize">
         </activity>
         <activity
-            android:name="org.commcare.activities.UnrecoverableErrorActivity"
-            android:theme="@style/Theme.Transparent">
```

Comment:
Fix for the black dialog issue
![screen](https://cloud.githubusercontent.com/assets/94817/16997607/2a0f04da-4e84-11e6-8473-f3f7978952dc.png)